src/main/resources/plp/packratRun.r

```diff
@@ -0,0 +1,33 @@
+packratRun <- function (packratPath, workDir, func){
```

Comment:
This and above files are quite similar to the PLE ones. Why not to create some base templates? (anyway we are using Mustache)